### DISCUSSION TITLE
Welcome to FSDS Discussions!

### DISCUSSION THREAD

**Discussion by davidoort** [FS-Online/Driverless-Competition-Simulator#290] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidoort** [comment]
I'll start! Hey everyone 👋, I'm David and I'm currently doing my Masters in Robotics at ETH Zurich. I was the autonomous software lead at Formula Student Team Delft in the 2019/2020 season, which is the season COVID hit us and we decided to create this simulator for FSOnline!

**Comment by SijmenHuizenga** [comment]
Oh hello there, I'm Sijmen, helped build this simulator for what feels like ages ago. I won't be doing much work here, but feel free to tag me if you need something changed to Github permissions :)

**Comment by wouter-heerwegh** [comment]
Hi, I'm Wouter, I was part of the electronics and driverless departments at Formula Electric Belgium (2019-2021). I'm currently still supporting my team where possible and sometimes I try to help people on this repo.

**Comment by mateusz-lichota** [comment]
Hi, I'm Mateusz, the captain of Imperial Driverless, and I've been recently doing some serious development on this simulator to adapt it for convenient use.

**Comment by b1n-ch1kn** [comment]
Hey, I'm Alastair, Driverless Team Lead at QUT Motorsport. We've been using this simulator for around 2 years now to aid our development and I'm hoping I can help support new teams that discover it by contributing new features and wider device/OS compatibility.
